app/views/users/shared/_error_messages.html.slim

```diff
@@ -2,9 +2,7 @@
   #error_explanation
 
   h2
-    = I18n.t("errors.messages.not_saved",
-    -                  count: resource.errors.count,
-    -                  resource: resource.class.model_name.human.downcase)
+    = I18n.t("errors.messages.not_saved",count: resource.errors.count,resource: resource.class.model_name.human.downcase)
```

Comment:
Vajag atstarpes pēc komata!

Reply:
es izlasiju par spaces , whitespaces , newlines , un to pirmo nodalu , bet sis viss kas palicis ir pirms es to biju izlasijis , tagad kad rakstu es censos pieverst uzmanibu kur lieku atstarpes.